src/firebase.js

```diff
@@ -0,0 +1,17 @@
+const firebase = require('firebase/app');
+require('firebase/auth');
+
+const firebaseConfig = {
+  apiKey: process.env.FIREBASE_API_KEY,
+  authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.FIREBASE_PROJECT_ID,
+  storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID,
```

Comment:
I don't think we're using anything that requires this messaging sender id value?

Reply:
Removed, I just copied the whole document from project settings.